tests/test_advisory_mgr.py

```diff
@@ -54,3 +54,8 @@ def test_get_dependent_advisories(self):
         self.assertTrue(ad.has_dependency())
         self.assertEqual(ad.get_dependent_advisories()[
                          0].errata_id, dependent_ad_id)
+
+    def test_check_ad_state(self):
```

Comment:
suggest to add unit test to cover scenario:
1. get advisory list
2. check dropped ad is not in the list.

Reply:
you can init 4.13.x config store (which has dropped ad) in your unit test case.

Reply:
please don't change the config store version in func `setUp`, it will impact other unit test. because the buid data is different. my suggestion is new a AdvisoryManager with corresponding config store version in your unit test func.

Reply:
Thank you @rioliu-rh. Corrected and pushed